Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyRomanNumeralHelper.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.helper.metadata.legacytypeimplementations;
+
+import org.kitodo.api.ugh.RomanNumeralInterface;
+import org.kitodo.production.helper.metadata.RomanNumeral;
+
+/**
+ * Connects the legacy Roman numeral to a Roman numeral implementation. This is
+ * a soldering class to keep legacy code operational which is about to be
+ * removed. Do not use this class.
+ */
```

Comment:
remove-todo is missing

Reply:
The class can not be removed before all places in the code where it is used have been re-implemented.